### PR TITLE
Generalizing docs

### PR DESCRIPTION
**Pull request by Pezi777**

### CODE REVIEW

File: user/tutorial.md [hunk 1]
```diff
@@ -14,16 +14,54 @@ instead.
 
 To start using Travis CI, make sure you have:
 
- * A [GitHub](https://github.com/) account.
- * Owner permissions for a project [hosted on GitHub](https://help.github.com/categories/importing-your-projects-to-github/).
+ * A [GitHub](https://github.com/) or [Bitbucket](https://bitbucket.org/) account.
+ * Owner permissions for a project hosted on [GitHub](https://help.github.com/categories/importing-your-projects-to-github/) or [Bitbucket](https://confluence.atlassian.com/bitbucketserver/importing-code-from-an-existing-project-776640909.html).
```
Comment (AlonaHlobina) [inline user/tutorial.md]: This link https://confluence.atlassian.com/bitbucketserver/importing-code-from-an-existing-project-776640909.html is for the server version of Bitbucket. I would rather link to this more generic page https://confluence.atlassian.com/bitbucket/repository-setup-877174034.html

Comment (AlonaHlobina) [reply user/tutorial.md]: Or maybe even this one https://confluence.atlassian.com/bitbucket/teams-321853005.html ?

File: user/travis-ci-for-private.md [hunk 2]
```diff
@@ -61,48 +60,31 @@ Travis CI is, and always will be, free for open source projects.
 For a list of plans and prices for private repositories, look at
 [travis-ci.com/plans](https://travis-ci.com/plans).
```
Comment (AlonaHlobina) [inline user/travis-ci-for-private.md]: For a list of plans and prices for private repositories, look at
[travis-ci.com/plans](https://travis-ci.com/plans) for not authorized users.
For a list of plans and prices for private repositories, look at
 [travis-ci.com/account/subscription](https://travis-ci.com/account/subscription) for authorized users

File: user/tutorial.md (hunk 1 above)
Comment (Pezi777) [reply user/tutorial.md]: I'll change it to this one: https://confluence.atlassian.com/bitbucket/transfer-repository-ownership-289964397.html

File: legal/security.md [hunk 3]
```diff
@@ -60,6 +60,25 @@ However, to allow us to automatically specify SSH keys, service hook configurati
...
 We only manually access your code when explicitly requested by you and only in explicit consent with you, and only to debug and help solve build issues.
 
+### How does Travis access my Bitbucket account?
+When you sign up for Travis, we collect an OAuth token from Bitbucket, which allows us to request data from the Bitbucket API on your behalf. This OAuth token is stored securely in our database and is protected from unauthorized access.
+
+The token is bound to permissions set on Bitbucket, so please make sure you've read their documentation on access control and API access permissions.
+
+We use this token in these situations, and under no other circumstances than described below.
+
+- To synchronize the repositories you have access to. We use this information to show you the available repositories on your profile page so you can enable or disable building them on Travis.
```
Comment (weban) [inline legal/security.md]: We have to add 'To  synchronize the teams...

File: legal/security.md [hunk 4]
```diff
@@ -72,6 +91,13 @@ These notifications don't include any sensitive information other than commit re
 
 We store these build notifications for debugging purposes, and for debugging purposes only.
 
+### What data do we store from Bitbucket?
+When you push code to Bitbucket for a repository that is set up to run on Travis, we get a push notification. The same is true for pull requests that are sent to us.
```
Comment (weban) [inline legal/security.md]: What about user info?

File: legal/terms-of-service.md [hunk 5]
```diff
@@ -52,7 +52,7 @@ as of 05/2018
 
 1. Adjustments, changes and updates of the Travis CI Software that help to avoid or maintain dysfunctions of the Travis CI Software may lead to temporary service suspensions. Travis CI will try to limit downtime of the service or restrictions of accessibility to 10 hours a month. Travis CI will try to do regular maintenance works during the weekend or at times between 10 p.m. and 6 a.m. (CET). Outside of the said hours, Travis CI will inform the Customer about upcoming maintenance works.
 2. The Customer is aware that the service relies on a working internet infrastructure. Additional downtime of the service can occur if the website is not available and at any other time with restrictive access to the internet of the Customer.
-3. The Customer is aware that the Travis CI Software does not work if GitHub is not properly available (be it to Travis CI or the Customer).
+3. The Customer is aware that the Travis CI Software does not work if GitHub/Atlassian Bitbucket is not properly available (be it to Travis CI or the Customer).
```
Comment (weban) [inline legal/terms-of-service.md]: Im wondering, maybe we can replace Github/Atlassian with some generic name for example 'Proveider*.We are going to add Gitlab/assembla support soon so we will have to update it again

Comment (Pezi777) [reply legal/terms-of-service.md]: we have to wait for the legal department's decision on that, but good point :-)

File: legal/security.md (hunk 4 above)
Comment (Pezi777) [reply legal/security.md]: please add everything you think is necessary

File: legal/security.md (hunk 3 above)
Comment (Pezi777) [reply legal/security.md]: please add this information directly in the file in the branch

**Review by EmmStewie2709 — COMMENTED** [review]
n p

**Pull request merged**